### ISSUE TITLE
Facing issues while trying to run code after cloning

### ISSUE THREAD

**Issue by Vibhash10** [man-group/dtale#567]
I have cloned the code and installed all the front-end dependencies by doing "npm install". Then I followed along with the instructions given in the cloning section about running the command "npm run watch" but when I do that the process is getting stuck after a few seconds after building dtale_bundle.js.

I want to add some custom functionalities to the code and then I want to run it on my local machine on a specific port. I request you to kindly provide me some help about how to run the code on my local machine. So that I can play with the code and then run it on a specific port to see whether my changes work. A response from you  will be very helpful for me.
![vsCodeDtale](https://user-images.githubusercontent.com/69476998/133940380-143bdbbf-dab0-4132-8f39-6c29c1c03009.PNG)

**Comment by aschonfeld** [comment]
@Vibhash10 sorry for the wait, based on your screenshot `npm run watch` is working correctly.  All that command does is compile the JS for you and then watch for any changes you make to the JS files and recompile it on the fly.  So Ideally you'll run `npm run watch` in one terminal and then in another terminal you'll start your python D-Tale process.

**Comment by Vibhash10** [comment]
Hello sir, as you told earlier that I would need to start my D-Tale python process on another terminal. I tried doing the same by referring to the cloning section and I executed the "python setup.py develop" command. I am not good with backend at all and I am getting the error as shown in the screenshot below. Any suggestion on how to solve this error. Eagerly waiting for your response sir.
![dtale_python_issue](https://user-images.githubusercontent.com/69476998/136103914-4f5b5343-746d-4d87-a91a-57b58a3b6b0d.PNG)
